duckduckgo/iOS
# Better reflect temporary unprotected status

Task/Issue URL: https://app.asana.com/0/0/1200517965702748/f
Tech Design URL:
CC: @brindy @bwaresiak 

**Description**:
This PR better reflects unprotected status for sites on the temporary allow list. A disclaimer has been added to inform users of this state.

**Steps to test this PR**:
1. Visit https://spiegel.de
2. Open privacy dashboard
1. Site should appear unprotected. The toggle should be hidden with a message informing the site is temporarily unprotected



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
